todo_app/data/trello_items.py

```diff
@@ -0,0 +1,114 @@
+from todo_app.data.todo_items import Item
+import requests, os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
You should find you don't need to explicitly call this, since `poetry run flask run` should pick up the `.env` file directly, but if you were testing this file directly (e.g. with `python todo_app/trello_items.py`) then this would be required